styles/main.css

```diff
@@ -130,33 +139,44 @@ header li {
 header nav {
     float: right;
     width: auto;
-    display: inline-block;
     margin: -4px 2em 0;
     padding: 0 2em;
     border: 1px rgba(0, 0, 0, 0.3);
-    border-bottom-style: solid;
+    border-bottom-style: dotted;
+    overflow: auto;

```

Comment:
This is the declaration that causes the nav element to take on the height of its floated contents